akd/src/utils.rs

```diff
@@ -45,10 +45,6 @@ pub(crate) fn empty_node_hash() -> crate::Digest {
     akd_core::hash::merge(&[akd_core::hash::hash(&EMPTY_VALUE), EMPTY_LABEL.hash()])
```

Comment:
nit: Like above, it looks like we could do the same here with `crate::hash...` instead of `akd_core::hash...`